### DISCUSSION TITLE
Outdated scores in leaderboard?

### DISCUSSION THREAD

**Discussion by AuracleTech** [ppy/osu#23744] [Q&A] [answered]
The guy right behind me has 863x combo even though max is 567x 😀 what ❔

![image](https://github.com/ppy/osu/assets/40531871/25f07c4a-ab9d-43eb-80a8-00ba9bebcf41)

me
![image](https://github.com/ppy/osu/assets/40531871/b2cd8504-488d-4a2e-aac0-30aece999473)

them
![image](https://github.com/ppy/osu/assets/40531871/4ef6c652-3c42-45d6-a57c-1f800a2440cb)

**Comment by integerrr** [comment, marked as answer]
They had classic mod on.

**Reply by AuracleTech** [reply]
Didn't know it would display combo differently

**Reply by integerrr** [reply]
Slider ends give combos in osu!stable and by extension, classic mod
